examples/Choropleth.ipynb

```diff
@@ -51,6 +52,46 @@
     "m"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# To add callback for style\n",
+    "def compute_style(feature, colormap, choro_data):\n",
+    "    return {\n",
+    "        'fillColor': colormap(choro_data),\n",
+    "        'weight': random.randint(1, 5),\n",
+    "    }"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m.remove_layer(layer)\n",
```

Comment:
If you fix the shadowing of `Choropleth._update_data`, you should be able to dynamically change the `style_callback`. So no need to remove the layer and add it back.

You should be able to write:
```python
layer.style_callback = compute_style
```
And that would work.

Reply:
Removed the statement